common/recordings.md

```diff
@@ -139,3 +140,39 @@ If a storage destination is not configured for recordings and if you choose to r
 | Stream ID   | Unique identifier for a particular stream of a room (audio-video/screenshare)                                                                                                               |
 | Track ID    | Unique identifier for a particular track (audio or video) of a stream                                                                                                                       |
 | Layer Index | Layer index values show descending HLS resolutions - 0(1080p), 1(720p), 2(480p), 3(360p) and 4(240p). If highest resolution of template is 720p, then 0(720p), 1(480p), 2(360p) and 3(240p) |
+
+## Chat Recording
+
+Chat recording is a feature through which you will receive all chats messages sent by peers during the SFU/browser recording. Chat recording is available for both SFU recording and browser recording. Only public chats sent to all roles will be recorded. The `.csv` file will be uploaded to the recording bucket configured for your video recordings. Chat recording is currently enabled from backend for few customers. The file header will be: `SenderPeerID,SenderName,SenderUserID,Roles,SentAt,Type,Message`
```

Comment:
> Chat recording is currently enabled from backend for few customers.
is this still true? can we change this before release?

Reply:
We have enabled it for few customers and yesterday we tried 12% customers - found an issue which will be fixed today then we will roll it out to all customers except classplus. This sentence can be removed but classplus can see that it's disabled for them so better to write its enabled for few customers or something?

Reply:
good point. i think it's ok if it's just 1. let's remove the sentence

Reply:
done removed.